test/es6/rlexe.xml

```diff
@@ -751,6 +751,13 @@
   </test>
   <test>
     <default>
+      <files>generators-deferparse.js</files>
+      <compile-flags>-force:deferparse -ES6Generators -JitES6Generators -ES6Classes -ES6DefaultArgs -args summary -endargs</compile-flags>
```

Comment:
Above test did not use the unittestframework - so passing -args summary -endargs is not useful. How about you have this test in the generators.js itself and call that in force deferparse mode as well?